geopyspark/geotrellis/catalog.py

```diff
@@ -161,6 +162,50 @@ def _construct_catalog(geopysc, new_uri, options):
                                           value_reader=value_reader,
                                           writer=writer)
 
+def read_metadata(geopysc,
```

Comment:
Similar function on `TiledRasterRDD` is `def layer_metadata(self)` so this should be `def read_layer_metadata` to keep it consistent.